iaf-commons/pom.xml

```diff
@@ -60,6 +60,16 @@
 			<artifactId>jsr305</artifactId>
 			<version>3.0.2</version>
 		</dependency>
+		<dependency>
+			<groupId>org.yaml</groupId>
+			<artifactId>snakeyaml</artifactId>
+			<version>2.2</version>
```

Comment:
Do you want to always add in this dependency, or do you want to make it provided / optional?